src/ocamlorg_frontend/pages/install.eml

```diff
@@ -120,12 +120,11 @@ Layout.render
         </h2>
         <ol>
           <li>
-            <h3>Install the DkML Windows distribution</h3>
+            <h3>Install the opam package manager</h3>
               <p>
-                The installer sets up OCaml 4.14.0 and <a target="_blank" href="https://github.com/ocaml/opam">OCaml's package manager opam</a>,
-                with Git and the Visual Studio compiler.
+                OCaml's package manager opam's 2.2 version onwards supports Windows natively. It is the recmmended way to install OCaml on Windows machines.
 
-                If you need a different OCaml version, check out the alternative installation instructions
+                If you are looking for a different installation method, check out the alternative installation instructions
                 provided in the <a href="<%s Url.tutorial "ocaml-on-windows" %>">"OCaml on Windows" guide</a>.
               </p>
 

```

Comment:
I can't suggest it in the GitHub UI, but this paragraph below can go (it's incorrect), cf. https://github.com/dra27/ocaml.org/commit/98255c3f94b8cb14e3aebcfd2aa5ac64a5885be3